src/MovieDetails/MovieDetails.css

```diff
@@ -0,0 +1,39 @@
+section {
+    height: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    background:  linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url('https://image.tmdb.org/t/p/original//x4UkhIQuHIJyeeOTdcbZ3t3gBSa.jpg') no-repeat center center fixed;
+    background-size: cover;
+}
+
+h1 {
+    font-size: 45px;
+    margin: 5px 0px 5px 0px;
+}
+
+h1, p {
+    text-shadow: .5px .5px black ;
+}
+
+.main-info {
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+}
```

Comment:
We probably want to make our font size bigger for the movie details, title and summary